release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -0,0 +1,573 @@
+[id="ocp-4-9-release-notes"]
+= {product-title} {product-version} release notes
+include::modules/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid cloud application platform for deploying both new and existing applications on secure, scalable resources with minimal configuration and management overhead. {product-title} supports a wide selection of programming languages and frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+
+Built on {op-system-base-full} and Kubernetes, {product-title} provides a more secure and scalable multi-tenant operating system for today's enterprise-class applications, while delivering integrated application runtimes and libraries. {product-title} enables organizations to meet security, privacy, compliance, and governance requirements.
+
+[id="ocp-4-9-about-this-release"]
+== About this release
+
+// TODO: Update k8s link once there is a version-specific URL for the Kubernetes release
+{product-title} (link:https://access.redhat.com/errata/RHSA-2021:2438[RHSA-2021:2438]) is now available. This release uses link:https://kubernetes.io/docs/setup/release/notes/[Kubernetes 1.21] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
+
+//Red Hat did not publicly release {product-title} 4.9.0 as the GA version and, instead, is releasing {product-title} 4.9.TBD as the GA version.
+
+{product-title} {product-version} clusters are available at https://cloud.redhat.com/openshift. The {cloud-redhat-com} application for {product-title} allows you to deploy OpenShift clusters to either on-premise or cloud environments.
+
+{product-title} {product-version} is supported on {op-system-base-full} 7.9 or later, as well as on {op-system-first} 4.8.
```

Comment:
Should 4.8 be 4.9 here?

Reply:
I think it will be, but the other version info here will also need an update. Leaving it for a full treatment from someone who knows everything :wink: